README.md

```diff
@@ -43,7 +43,7 @@ RDMA and GPUDirect RDMA workloads in a kubernetes cluster including:
 * Kubernetes secondary network for Network intensive workloads
 
 ## Documentation
-For more information please visit the official [documentation](https://docs.nvidia.com/networking/display/kubernetes2310/Network+Operator).
+For more information please visit the official [documentation](https://docs.nvidia.com/networking/display/kubernetes2410).
```

Comment:
Should we use just a most general landing page, listing the versions? It will save us updating this each release.

https://docs.nvidia.com/networking/software/cloud-orchestration/index.html

Reply:
I'm not sure how the routing works for these - but could we just have the base page point to latest? A first change would be to make changing this part of the release task so that we don't have to do it in a separate PR.

Reply:
LGTM. We'll probably update in to some 'latest' version once we'll switch to new documentation process

Reply:
Not sure about having routing from latest. We should check with technical writers team about it.